devtools/gradle/gradle-application-plugin/src/main/java/io/quarkus/gradle/tasks/QuarkusInfo.java

```diff
@@ -34,6 +34,10 @@ public QuarkusInfo() {
 
     @TaskAction
     public void logInfo() {
+
+        getProject().getLogger()
+                .warn("This task is experimental, its options and output will be changing in the upcoming versions");
```

Comment:
```suggestion
                .warn("This task is experimental, its options and output might change in future versions");
```
? The `will` seems a bit too assertive to me. Same for all the other messages.

Reply:
Applied, thanks!